witx/http-handler.witx

```diff
@@ -2,11 +2,41 @@
 ;;
 ;; The HTTP handler ABI defines functions invoked on an HTTP server handling
```

Comment:
ps it would be cool to have this as the top of the http-handler.md, just not sure the magic to do that.